Kitodo/src/main/java/de/sub/goobi/export/dms/ExportDms.java

```diff
@@ -484,8 +494,7 @@ public void imageDownload(Process process, URI userHome, String atsPpnBand, fina
                 }
             } else {
                 /*
-                 * wenn kein Agora-Import, dann den Ordner mit
-                 * Benutzerberechtigung neu anlegen
+                 * wenn kein Agora-Import, dann den Ordner mit Benutzerberechtigung neu anlegen
```

Comment:
As you or your IDE is changing this, please translate it to English and maybe change it to a one line comment style. As mentioned before: it is more a asynchronous task and not "Agora"